Split manifest generation to produce both v1/v1beta1 CRDs (#4489)
elastic/cloud-on-k8s: docs/operating-eck/webhook.asciidoc

[id="{p}-{page_id}-defaults"]
== Defaults provided by ECK

> When installed through the default `operator.yaml` manifest, ECK will be installed with a `ValidatingWebhookConfiguration` configured as follows:

Reviewer: ```suggestion
When using the default `operator.yaml` manifest, ECK is installed with a `ValidatingWebhookConfiguration` configured as follows:
```
